Makefile

```
@@ -43,6 +46,7 @@ lint: clean
 
 # used in development
 dev-setup:
+	$(VENV)
```

This 👇 is the correct form of "dependent"

```suggestion
dev-setup: $(VENV)
```

---

Oh, I see. Corrected.